src/lidar_object_recognition_2/scripts/DBSCAN_variable.txt

```diff
@@ -0,0 +1,2 @@
+20
```

Comment:
What are these numbers for?

Reply:
Those are similar to the old library work, those number represent the minimum samples and epsilon for cluster detection

Reply:
Ok. We might want to move this into a config class with other constants in the future.